app/configurator/components/add-dataset-dialog.tsx

```diff
@@ -156,6 +171,54 @@ const NewAnnotation = (props: TypographyProps) => {
   );
 };
 
+const useCautionAlert = () => {
+  const [isOpen, setIsOpen] = useLocalState<boolean>(
```

Comment:
Nit: I think you don't have to use `<boolean>`, as the type should be inferred by looking at the implementation of `useLocalState`?